src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -146,7 +146,7 @@ def get_hosts(sources=None):
             hosts = [
                 merge_dicts(
                     {
-                        'ceph_version': '',
+                        'ceph_version': mgr.version,
```

Comment:
This will be the version of the current ceph-mgr, not from the packages installed on a specific node. Is that the expected behaviour?

Reply:
In the previous implementation [this](https://github.com/ceph/ceph/pull/52154/files) was only used. 
Are we using anywhere that in the dashboard as most of the places I see `mgr.version` is being used to report ceph version. Also how to fetch the version from specific node ? And what is the difference ? 
This was a downstream issue where user reported that he was able to see version earlier and that now its missing. Thanks